test/contract.js

```diff
@@ -0,0 +1,109 @@
+import 'babel-polyfill';
```

Comment:
What does this line do? Does it change the way we run test
"mocha --compilers js:babel-register " ?

Reply:
This allows me to use async/await with babel.